pkg/operator/ceph/config/defaults.go

```diff
@@ -67,6 +67,13 @@ func DefaultCentralizedConfigs(cephVersion version.CephVersion) []Option {
 		}...)
 	}
 
+	// For Octopus
+	if cephVersion.IsAtLeastOctopus() {
+		overrides = append(overrides, []Option{
+			configOverride("global", "mon allow pool size one", "true"),
```

Comment:
Can we just add this overrides without checking the version? Nautilus would ignore settings it doesn't understand, right?

Reply:
Nop, the store will report an error for an unknown option.